src/main/java/witchinggadgets/common/WGContent.java

```diff
@@ -186,13 +186,23 @@ public static void preInit() {
         preInitItems();
         preInitBlocks();
     }
+
+    private static boolean clusterBlacklist(Materials material) {
+        return material.equals(Materials.Iron) || material.equals(Materials.Copper)
```

Comment:
Looks fine

Reply:
I wanted it like that if I ever have to hardcode the removal of some other material.

Reply:
Yeah I completely understand